site/src/pages/release-notes/1.17.2.mdx

```diff
@@ -0,0 +1,15 @@
+---
+date: 2022-08-02
+---
+
+## 🛠️ Bug fixes
+
+- Cors requests with query params work correctly. #4378
```

Comment:
Would it help to also specify since what version this bug was introduced?

```suggestion
- CORS requests with query params are handled correctly. #4378
    - Armeria versions from 1.15.0 to 1.17.0 are affected.
```

Reply:
Looks nice.